backend/src/Manager/User/UserManager.php

```diff
@@ -114,4 +114,15 @@ public function getUserRoleByUserId($userId): ?array
     {
         return $this->userRepository->getUserRoleByUserId($userId);
     }
+    public function checkUserType($userType,$userID): string
```

Comment:
I think the whole API not needed any more but u can keep it for now but must add docs